worldedit-core/src/main/java/com/sk89q/worldedit/EditSession.java

```diff
@@ -375,14 +408,40 @@ public void setExtent(AbstractDelegateExtent extent) {
         new ExtentTraverser(this).setNext(extent);
     }
 
+    //FAWE Start
+
     /**
-     * Get the Player or null.
+     * Get the Actor or null.
      *
-     * @return the player
+     * @return the actor
      */
     @Nullable
-    public Player getPlayer() {
-        return player;
+    public Actor getActor() {
+        return actor;
+    }
+    //FAWE End
+
+    private Extent traceIfNeeded(Extent input) {
+        Extent output = input;
+        if (tracingExtents != null) {
+            TracingExtent newExtent = new TracingExtent(input);
+            output = newExtent;
+            tracingExtents.add(newExtent);
+        }
+        return output;
+    }
+
+    private Extent wrapExtent(Extent extent, EventBus eventBus, EditSessionEvent event, Stage stage) {
```

Comment:
These methods don't appear to be used anywhere? It appears our EditSession looks a whole different to upstream ones.